spec/mailers/requests_confirmation_mailer_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:request) { create(:request, organization:, partner_user:) }
   let(:mail) { RequestsConfirmationMailer.confirmation_email(request) }
 
-  let(:request_w_varied_quantities) { create(:request, :with_varied_quantities, organization: organization) }
+  let(:request_w_varied_quantities) { create(:request, :with_varied_quantities, :with_item_requests, organization: organization) }
```

Comment:
A lot of these factories now require `:with_item_requests` role; in a further PR I'll make that the default and then remove all of the instances of the param.